src/getjump/Vk/Wrapper/Photos.php

```diff
@@ -56,18 +62,28 @@ public function saveMessagesPhoto($data)
             ->request('photos.saveMessagesPhoto')->one();
     }
 
-    public function uploadAlbum(array $files = array(), $album = false, $group = false)
+    /**
+     * Uploading photos in album
+     *
+     * @param array $listOfFilePaths array with file paths, which you want to upload to an album
+     * @param bool|int|string $album_id ID of album. Album for uploading files
+     * @param bool|int|string $group_id ID of group. Group which belongs album (optional)
+     */
+    public function uploadAlbum(array $listOfFilePaths = [], $album_id = false, $group_id = false)
     {
-        if (sizeof($files) > 5 || sizeof($files) == 0) {
+        if (sizeof($listOfFilePaths) > 5 || sizeof($listOfFilePaths) == 0) {
```

Comment:
Looks more complicated than it is. `$files` was good, just because phpDoc giving all necessary information.